src/Neo/SmartContract/Native/CryptoLib.cs

```diff
@@ -115,5 +140,67 @@ public static bool VerifyWithECDsaV0(byte[] message, byte[] pubkey, byte[] signa
                 return false;
             }
         }
+
+        /// <summary>
+        /// Recovers the public key from a secp256k1 signature in a single byte array format.
+        /// </summary>
+        /// <param name="message">The original message that was signed.</param>
+        /// <param name="hasher">The hash algorithm to be used (SHA256 or Keccak256).</param>
+        /// <param name="signature">The 65-byte signature in format: r[32] + s[32] + v[1]. 64-bytes for eip-2098, where v must be 27 or 28.</param>
+        /// <returns>The recovered public key in compressed format, or null if recovery fails.</returns>
+        [ContractMethod(Hardfork.HF_Echidna, CpuFee = 1 << 10, Name = "secp256k1Recover")]
+        public static byte[] Secp256K1Recover(byte[] message, Hasher hasher, byte[] signature)
```

Comment:
```suggestion
        public static byte[] RecoverSecp256K1(byte[] message, Hasher hasher, byte[] signature)
```

Please change the name. Not proper grammar for english. We describe something 1st before saying what that thing is.